src/datasources/workspace/WorkspaceDataSource.test.ts

```diff
@@ -0,0 +1,36 @@
+import { BackendSrv } from '@grafana/runtime';
+import { MockProxy } from 'jest-mock-extended';
+import _ from 'lodash';
+import {
+  createFetchError,
+  createFetchResponse,
+  requestMatching,
+  setupDataSource,
+} from 'test/fixtures';
+import { WorkspaceDataSource } from './WorkspaceDataSource';
+
+let ds: WorkspaceDataSource, backendSrv: MockProxy<BackendSrv>;
+
+beforeEach(() => {
+  [ds, backendSrv] = setupDataSource(WorkspaceDataSource);
+});
+
+fdescribe('testDatasource', () => {
```

Comment:
```suggestion
describe('testDatasource', () => {
```

Reply:
I should see if I can update the linting rules to catch these

Reply:
good catch